webserver/views/api.py

```diff
@@ -188,7 +189,7 @@ def _messybrainz_lookup(listens):
         msb_listens.append(messy_dict)
 
     try:
-        msb_responses = messybrainz.submit_listens_and_sing_me_a_sweet_song(msb_listens)
+        msb_responses = messybrainz.submit_listens(msb_listens)
```

Comment:
this is one of the reasons that I like absolute package names - if here it said `webserver.external.messybrainz` it would be a lot more explicit about what we were calling. As it is, someone who glances at this code could think that this method is in the `messybrainz` package.